docs/src/content/docs/utilities/add-devtoolbar-plugin.mdx

```diff
@@ -0,0 +1,301 @@
+---
+title: addDevToolbarPlugin
+description:  allows you to register a framework component as a Dev Toolbar Plugin.
+---
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+`addDevToolbarPlugin` allows you to register a framework component as a Dev Toolbar Plugin!
+
+So now you can use a React, Preact, Solid, Vue or Svelte component as a Dev Toolbar Plugin,
+instead of having to create your DOM manually yourself!
```

Comment:
```suggestion
`addDevToolbarPlugin` allows you to register a framework component as a Dev Toolbar Plugin! You can now use a React, Preact, Solid, Vue or Svelte component instead of manipulating the DOM imperatively!
```